### ISSUE TITLE
tk_ts causes deprecated warning

### ISSUE THREAD

**Issue by Someone894** [business-science/timetk#52]
Hello,
since the `dplyr` 0.7.0 update the `timetk::tk_ts` function causes a deprecated warning.
Here is a minimal working example:

```
timetk::tk_ts(data.frame(
  A = 1:4,
  B = c(
    as.Date("1990-01-10"),
    as.Date("1990-02-10"),
    as.Date("1990-03-10"),
    as.Date("1990-04-10")
  )
),
select = "A"
)
```
which results in the actual result and:

```
Warning:
`select_()` is deprecated as of dplyr 0.7.0.
Please use `select()` instead.
This warning is displayed once every 8 hours.
Call `lifecycle::last_warnings()` to see where this warning was generated.
```

Like suggested I ran `lifecycle::last_warnings()` which resulted in:

```
<deprecated>
message: `select_()` is deprecated as of dplyr 0.7.0.
Please use `select()` instead.
This warning is displayed once every 8 hours.
Call `lifecycle::last_warnings()` to see where this warning was generated.
backtrace:
 1. timetk::tk_ts(...)
 3. timetk::tk_ts_.data.frame(...)
 7. timetk:::tk_xts_.data.frame(ret, select = select, silent = silent)
 8. dplyr::select_(data, select)
 9. dplyr:::lazy_deprec("select", hint = FALSE)
```
So it seems that [this](https://github.com/business-science/timetk/blob/230bd31e558895e7171628e0593aae48c4f8954f/R/coersion-tk_xts.R#L93) is the depreciated function.
Is this hard to fix? It would be lovely if you could use the new function and thus remove the warning.
Thank you very much :-)

**Comment by mdancho84** [comment]
This is going to be a pain to fix since I built everything back when it was cool to use select_()

Thanks for the heads up. Will tackle when I get some free time.

**Comment by Fuco1** [comment]
@mdancho84 maybe I don't get it but it seems there's only one use of `select_` in the entire package.  So changing just that one line should be enough.

**Comment by Fuco1** [comment]
As a workaround, I'm now pre-selecting the data frame/tibble before sending to the `tk_ts` function:

```R
data <- select(data, c("date", "quantity"))
data %>% tk_ts(frequency = 12, start = c(startYear, startMonth), silent = TRUE)
```

Make sure to pre-select the date-like column!

**Comment by mdancho84** [comment]
This is fixed in version 2.6.1
